src/main/groovy/de/dkfz/roddy/execution/jobs/JobManagerOptions.groovy

```diff
@@ -29,18 +29,56 @@ class JobManagerOptions {
     String userEmail
     String userMask
 
+    /**
+     * Request memory per job
+     */
+    boolean requestMemoryIsEnabled
+
+    /**
+     * Request walltime per job
+     */
+    boolean requestWalltimeIsEnabled
+
+    /**
+     * Request queue for a job
+     */
+    boolean requestQueueIsEnabled
+
+    /**
+     * Request cores per job (also effects node request)
+     */
+    boolean requestCoresIsEnabled
+
+    /**
+     * Request storage per job
+     */
+    boolean requestStorageIsEnabled
+
+    Map<String, String> additionalOptions
```

Comment:
Is this used anywhere?
I couldn't find any usage

Reply:
Yes it is. In the class above you'll find read access. Of course you have to enable things it's a parameter object. In Roddy we do it, in OTP you might or might not want to do it.